.github/workflows/deploy_mkdocs.yml

```diff
@@ -0,0 +1,24 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy MkDocs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docs/**'
+      - 'mkdocs.yml'
+
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - run: pip install -r mkdocs_requirements.txt
+      - run: mkdocs gh-deploy --force
```

Comment:
I believe this action will fail because it's pushing straight to main which violates org rules. 

Happy to leave for now and find out if it works. This is the action used in other repos: https://github.com/ONS-Innovation/github-copilot-usage-dashboard/blob/master/.github/workflows/deploy_mkdocs.yml

I know it's not as clean as `mkdocs gh-deploy` as it relies on a manual build, but it's a work around for the org rules.